tests/test_isrFunctions.py

```diff
@@ -104,6 +155,38 @@ def test_makeThresholdMask(self):
 
         self.assertEqual(len(defectList), 1)
 
+    def test_ITLEdgeBleedMask(self):
+        """Expect number of masked pixels according to edge bleed masking.
+        """
+        ITLdetector = TestITLDetector()
+        ITLexposure = TestITLExposure(ITLdetector.getBBox())
+        ITLexposure.setDetector(ITLdetector)
+
+        saturationLevel = 12000.
+        ITLexposure.fillMetadata(saturationLevel)
+        # Approximate saturated star
+        ITLexposure.image.array[1800-70:1800+70, 1100-40:1100+40] = 15000.
+        ITLexposure.mask.array[:, :] = 0
+
+        # Approximate edge bleed at bottom amp
+        ITLexposure.image.array[30:1800, 1100-3:1100+3] = 15000.
+        for y in range(30):
+            edgeBleedWidth = 50*np.exp(-0.05*y)+8
+            ITLexposure.image.array[y, 1100-int(edgeBleedWidth/2.)] = 15000.
+            ITLexposure.image.array[y, 1100+int(edgeBleedWidth/2.)] = 15000.
+            ITLexposure.image.array[y,
+                                    1100-int(edgeBleedWidth/2.)+1:1100+int(edgeBleedWidth/2.)
+                                    ] = 0.85*saturationLevel
+
+        badAmpDict = {}
+        for amp in ITLexposure.getDetector():
+            badAmpDict[amp.getName()] = False
+        ipIsr.maskITLEdgeBleed(ITLexposure, itlEdgeBleedSatMinArea=10000.,
+                               itlEdgeBleedSatFracLevel=0.8, itlEdgeBleedModelConstant=0.05,
+                               saturatedMaskName='SAT', badAmpDict=badAmpDict)
+
+        self.assertEqual(ipIsr.countMaskedPixels(ITLexposure, 'SAT'), 2298)
```

Comment:
This needs some tests to make sure (a) there are saturated pixels at each edge; (b) I guess there was no original SAT mask set?